src/sensors/scripts/segmented_image.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
python3

Reply:
Also you should be able to combine this with car_image_raw.py

Reply:
Ah, didn't realize the init files would mess with ROS, didn't see that on my end, will fix. As for putting it in car_image_raw.py its in a separate folder for that justification you can replace it with a ml model later on more easily, just part of that shaky justification.